tests/core/arch-arm64/stalker-arm64.c

```diff
@@ -1240,6 +1242,25 @@ TESTCASE (follow_thread)
 #endif
 }
 
+TESTCASE (pthread_create)
+{
+  fixture->sink->mask = (GumEventType) (GUM_COMPILE);
+
+  gum_stalker_follow_me (fixture->stalker, fixture->transformer,
+      GUM_EVENT_SINK (fixture->sink));
+
+  pthread_t thread;
+  int number = 0;
+  int ret = pthread_create (&thread, NULL, increment_integer, (gpointer) &number);
```

Comment:
Variables should be declared at the beginning of the scope, ordered by first use, with one blank line between them and the first statement. (And if one of them is used to store the return value of the function, it should be first.)

Reply:
Done